composer.json

```diff
@@ -22,10 +22,10 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "~7.4.0 || ~8.0.0 || ~8.1.0"
```

Comment:
I know it's too late to be restrictive on the upper bound, but better late then never, no?

Reply:
Aye: sad truth about `php-src`, heh.

@Potherca just FYI, PHP breaks compatibility in every damn minor release.

Reply:
Yeah. I've been through a few of those... been developing in PHP since 2002... 😢